ostia-operator/test/tests/01-simple_endpoint/run.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -e
+
+source ../common/common.sh
+
+ns="simple-api-$(generate_random_string)"
+endpoint="${ns}.${OPENSHIFT_PUBLIC_HOSTNAME}"
+
+echo "Testing Simple API Endpoint in namespace ${ns}"
+
+oc new-project ${ns} > /dev/null
```

Comment:
this prologue (and maybe even a bit more things that come after this point up to when waiting for pods) could be automated by calling some sort of setup function

Reply:
Thats a fair point. But its working on the assumption that we will always need an operator(obviously), apicast and and endpoint. Can you think of scenario's where either of the latter two would not be required? I can't